debug/gdbserver.py

```diff
@@ -887,7 +887,8 @@ def test(self):
 class MemorySampleMixed(MemorySampleTest):
     def test(self):
         addr = {}
-        for i, name in enumerate(("j", "i32", "i64")):
+        test_vars = ("j", "i32") + ("i64",) * (self.hart.xlen >= 64)
```

Comment:
Just to make the code more clear:
```suggestion
        test_vars = ["j", "i32"]
        if self.hart.xlen >= 64:
            test_vars.append("i64")
```

Reply:
You're right, this is much clearer. I force-pushed this change.